packages/patternfly-4/react-core/src/components/Nav/NavList.js

```diff
@@ -3,24 +3,27 @@ import PropTypes from 'prop-types';
 import { NavVariants } from './NavVariants';
 import styles from '@patternfly/patternfly-next/components/Nav/nav.css';
 import { css } from '@patternfly/react-styles';
+import { PageContext } from '../../layouts/Page/Page';
 
 const propTypes = {
   /** Indicates the list type. */
   variant: PropTypes.oneOf(Object.values(NavVariants)),
   /** Children nodes */
   children: PropTypes.node,
   /** Additional classes added to the list */
-  className: PropTypes.string
+  className: PropTypes.string,
+  /** nav should have tall modifer */
```

Comment:
Can you make this comment more descriptive? Maybe something like: Horizontal nav only: If true makes the nav taller

Reply:
added comments